backend/src/main/java/mpd/MpdServiceImpl.java

```diff
@@ -39,7 +40,7 @@ public MpdServiceImpl(String server, int port) {
     public void isConnected() {
         logger.info("Configure MPD communicator [{}:{}]", server, port);
         if (!mpd.isConnected()) {
-            throw new RuntimeException(String.format("Can not establish connection to MPD server %s:%d", server, port));
+            throw new GeneralException(String.format("Can not establish connection to MPD server %s:%d", server, port));
```

Comment:
Wouldn't a `IllegalStateException` be more appropriate for this case? It is always a good idea to use well-known Exceptions, if possible. Furthermore, I don't see any benefit of using the `GeneralException`, since it does not provide more context about the issue and the class is missing documentation.

Reply:
Would be nice if we could review the current 'exception concept' - currently everything is wrapped in this GeneralException. Ill open a new PR for this.

Reply:
Fixed in e5f4f08e94f1932938e1a19648f0b7eaf5694f0d.